repos/application-insights.proj

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
   <PropertyGroup>
-    <PackagesOutput>$(ProjectDirectory)/bin/$(Configuration)</PackagesOutput>
+    <PackagesOutput>$(SubmoduleDirectory)/bin/$(Configuration)</PackagesOutput>
```

Comment:
What was this changed? I would expect ProjectDirectory to be the correct location. https://github.com/dseefeld/source-build/blob/a4c1360fa9c6af9638bf8ad8c5f434f0e911579f/repos/dir.props#L7

Reply:
Source-build wasn't capturing appInsights output because the appInsights repo builds its output into a bin directory as a peer to the source directory.  This change is pointing to that directory to pickup the AppInsights package.